capstone-backend/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -61,22 +68,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String profileImageUrl = (String) payload.get("picture");
 
         googleUser = new User(googleId, email, fullname, profileImageUrl,
-            new ImmutableMap.Builder<String, String>()
-              .put("token_id", tokenId)
-              .put("token_type", token_type)
-              .put("access_token", access_token)
-              .put("scope", scope)
-              .put("idpId", idpId)
-              .build());
+            new ImmutableMap.Builder<String, String>().put("token_id", tokenId).put("token_type", token_type)
+                .put("access_token", access_token).put("scope", scope).put("idpId", idpId).build());
```

Comment:
I actually find the previous version much more readable with each field on a separate line. I don't see anything about this in particular in the style guide but I've definitely seen that pattern in Java code across Google.

nit: it might be helpful to alphabetize the field names too, especially if this list might grow.

Reply:
Yeah, Gitpod formatted it that way for some reason. I will change it in my next PR.

Reply:
Ok, I reverted the Gitpod formatting changes and alphabetize the field names.